src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -186,7 +253,8 @@ await contextualFilter.EvaluateAsync(context, appContext).ConfigureAwait(false)
                         {
                             BindSettings(filter, context, filterIndex);
 
-                            if (await featureFilter.EvaluateAsync(context).ConfigureAwait(false) == targetEvaluation) {
+                            if (await featureFilter.EvaluateAsync(context).ConfigureAwait(false) == targetEvaluation)
+                            {
```

Comment:
There appears to be a lot of whitespace added to the end of this line.

Reply:
I can't seem to find any whitespace when I look at it locally,